includes/agentic-commerce/class-wc-stripe-agentic-commerce-order-mapper.php

```
@@ -450,9 +450,34 @@ private function map_shipping( WC_Order $order, WC_Stripe_Agentic_Checkout_Sessi
+			$rates    = $packages[0]['rates'] ?? [];
+		} catch ( Throwable $e ) {
+			WC_Stripe_Logger::warning(
+				'Agentic order mapper: WC shipping calculation failed; will use free-form shipping line.',
+				[
+					'session_id' => $session->get_id(),
+					'error'      => $e->getMessage(),
+					'exception'  => get_class( $e ),
+					'file'       => $e->getFile(),
+					'line'       => $e->getLine(),
+				]
+			);
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Logger' in method 'map_shipping'. (undefined)

(StaticAccess)